Jellyfin.Server.Implementations/Migrations/20210906152818_NewImageModel.cs

```diff
@@ -0,0 +1,217 @@
+                    nullable: true);
+
+                migrationBuilder.AddColumn<DateTime>(
+                    name: "FileModificationDate",
+                    schema: "jellyfin",
+                    table: "ImageInfos",
+                    type: "TEXT",
+                    nullable: true);
+
+                migrationBuilder.AddColumn<string>(
+                    name: "Blurhash",
+                    schema: "jellyfin",
+                    table: "ImageInfos",
+                    type: "TEXT",
+                    nullable: true);
+
+                migrationBuilder.DropPrimaryKey(
+                    name: "PK_ImageInfos",
+                    schema: "jellyfin",
+                    table: "ImageInfos");
+
+                migrationBuilder.AddColumn<Guid>(
+                    name: "UUID",
+                    schema: "jellyfin",
+                    table: "ImageInfos",
+                    type: "TEXT",
+                    nullable: true,
+                    defaultValue: new Guid("00000000-0000-0000-0000-000000000000"));
+
+                var triggerQuery = @"CREATE TRIGGER AutoGenerateGUID
+                                    AFTER UPDATE ON ImageInfos
+                                    FOR EACH ROW
+                                    BEGIN
+                                        UPDATE ImageInfos SET UUID = (select hex(randomblob(4)) || '-' || hex(randomblob(2))
+                                        || '-' || '4' || substr(hex(randomblob(2)), 2) || '-'
+                                        || substr('AB89', 1 + (abs(random()) % 4), 1) ||
+                                        substr(hex(randomblob(2)), 2) || '-' || hex(randomblob(6))) 
+                                        WHERE rowid = NEW.rowid;
+                                    END";
+
+                migrationBuilder.Sql(triggerQuery);
+
+                migrationBuilder.Sql("UPDATE ImageInfos SET RowVersion = 1");
+
+                migrationBuilder.Sql("DROP TRIGGER AutoGenerateGUID");
+
+                migrationBuilder.Sql("UPDATE Users SET RowVersion = (SELECT RowVersion+1 FROM Users u WHERE u.Id = Users.Id)");
+
+                migrationBuilder.RenameTable(
+                    name: "ImageInfos",
+                    schema: "jellyfin",
+                    newName: "Images",
+                    newSchema: "jellyfin");
+
+                migrationBuilder.AddPrimaryKey(
+                    name: "PK_Images",
+                    schema: "jellyfin",
+                    table: "Images",
+                    column: "Id");
+
+                migrationBuilder.Sql("DELETE FROM Images WHERE Id NOT IN (SELECT ImageId FROM Users)");
+
+                // End of the custom migration procedures
+            }
+            else
+            {
+                migrationBuilder.DropTable(
+                    name: "ImageInfos",
+                    schema: "jellyfin");
+
+                migrationBuilder.CreateTable(
```

Comment:
The library.db migration also introduced a ImageInfo table
https://github.com/jellyfin/jellyfin/blob/master/src/Jellyfin.Database/Jellyfin.Database.Implementations/Entities/BaseItemImageInfo.cs#L10
Could you repurpose that instead?